app/containers/EventCreationContainer.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { setName,
+    setAddress,
+    setDate,
+    setZipcode,
+    setManager,
+    setError,
+    removeError,
+    reset,
+    saveEvent} from '../actions/eventcreation';
+import { push as pushRoute} from 'react-router-redux';
+import EventCreation from '../components/EventCreation/EventCreation.js'
+
+
+class EventCreationContainer extends Component {
+
+    componentWillUpdate(nextProps, nextState)
+    {
+        if (nextProps.success) {
+            nextProps.pushRoute('/profile');
+        }
+    }
```

Comment:
this doesn't make sense in this context. Review react lifecycle methods - https://facebook.github.io/react/docs/component-specs.html#lifecycle-methods

Decide what you want to do once the event is successfully saved and before everything is re-rendered